src/App.js

```diff
@@ -22,9 +22,6 @@ import './App.css';
 import { signOutUser } from './utils/userAuth';
 import SprintSelectionComponent from './components/Sprints/SprintSelectionComponent';
 import SprintRetrospectiveComponent from './components/Retrospective/SprintRetrospectiveComponent';
-import "./App.css";
-import { signOutUser } from "./utils/userAuth";
```

Comment:
is .css really not being used at all?

Reply:
It was declared twice cause i did an oop with the merve